packages/filecmd.rb

```diff
@@ -38,7 +42,12 @@ def self.check
   end
   
   def self.install
-    system "install -Dm755 filefix #{CREW_DEST_PREFIX}/bin/filefix"
-    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install-strip"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin/"
+    FileUtils.install 'filefix', "#{CREW_DEST_PREFIX}/bin/filefix", mode: 0755
+    system "make DESTDIR=#{CREW_DEST_DIR} install-strip | sed -e \"s,libtool: warning: remember to run 'libtool --finish #{CREW_LIB_PREFIX}',,g\""
+  end
+  
+  def self.postinstall
+    system "libtool --finish #{CREW_LIB_PREFIX}"
```

Comment:
Does it make sense to add this in `crew`?

Reply:
I've seen lots of packages suggest that...

Reply:
I mean incorporate this logic in the `crew` command.